Algorithms/RunningSumArray.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Algorithms
+{
+    public class RunningSumArray
+    {
+        /// <summary>
+        /// Big O
+        /// Complexity: O(n) - The array is only traversed once, and the new value is the current position plus the value of the previous position, with accumulated value
+        /// </summary>
+        /// <param name="numsArray">Numbers array</param>
+        /// <returns>Return the running sum of numbers.</returns>
+        public int[] RunningSum(int[] numsArray)
```

Comment:
what do you think about this code now? Comparing it to the first version

Reply:
I think it's better, because by use the same array to overwrite value, i don't need more memory space. I also think it's cleaner to reduce the number of lines of code.